### PR TITLE
feat: browser-interface for windows and tutorial

### PR DESCRIPTION
**Pull request by Kinerius**
## What does this PR change?

Fixed the `Makefile` of `browser-interface` for it to work with Windows.

Added a tutorial to setup `browser-interface` using `Visual Studio Code` with `devcontainer` extension. 

## How to test the changes?

Follow the` browser-tutorial` setup-guide

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by menduz — APPROVED** [review]

**Review by dalkia — APPROVED** [review]
Nice to know that I can go back to Windows xD

**Comment by Kinerius** [comment]
> Nice to know that I can go back to Windows xD

Did it work for you? I need confirmation that this branch is working for everyone in windows.

@popuz 
@lorux0 
@0xD-Fabio

**Comment by dalkia** [comment]
> > Nice to know that I can go back to Windows xD
> 
> Did it work for you? I need confirmation that this branch is working for everyone in windows.
> 
> @popuz @lorux0 @0xD-Fabio

I stopped using Windows to avoid the `browser-interface` issues. Nonetheless, I gave it a shot, got as far as here by following the instructions

image

File: README.md [hunk 1]
```diff
@@ -59,6 +59,24 @@ To test against a build made on this repository, you can use a link with this fo
...
+## Running the browser-interface
+
+In order to run browser interface in any platform follow the next instructions
+
+### Steps
+
+1. Open browser-interface with `Visual Studio Code`
+2. Make sure you have the `devcontainers` extension installed
+3. Make sure [Docker Desktop](https://www.docker.com/) is running
+4. `At Visual Studio Code` you will find a green button at the bottom-left corner of your screen, if you press it, a menu will appear and you have to press `Reopen in Container`
```
Comment (popuz) [inline README.md:71]: `At Visual Studio Code` you will find a green button at the bottom-left corner of your screen (it has _"Open a Remote Window"_  tooltip),

**Pull request merged**